alacritty_terminal/src/input.rs

```diff
@@ -283,27 +283,33 @@ pub enum Action {
     #[serde(skip)]
     Command(String, Vec<String>),
 
-    /// Hides the Alacritty window
+    /// Hide the Alacritty window
     Hide,
 
-    /// Quits Alacritty.
+    /// Quit Alacritty
     Quit,
 
-    /// Clears warning and error notices.
+    /// Clear warning and error notices
     ClearLogNotice,
 
-    /// Spawn a new instance of Alacritty.
+    /// Spawn a new instance of Alacritty
     SpawnNewInstance,
 
-    /// Toggle fullscreen.
+    /// Toggle fullscreen
     ToggleFullscreen,
 
-    /// Toggle simple fullscreen on macos.
+    /// Toggle simple fullscreen on macos
     #[cfg(target_os = "macos")]
     ToggleSimpleFullscreen,
 
-    /// No action.
+    /// No action
```

Comment:
I'm not against unifying the documentation, that's actually much appreciated. But the canonical format for doc comments in Rust is to have the punctuation both in the description and the headline. So that should be kept.

Reply:
Eh, sorry. There were no dots for the most of the lines at the start, so I thought that's the standard.